### PR TITLE
Added types package

### PR DESCRIPTION
**Pull request by piotrpospiech**
- Added `zodern:types` package.
- Added default export for `i18n`

### CODE REVIEW

File: source/common.ts [hunk 1]
```diff
@@ -307,3 +307,4 @@ i18n.__ = i18n.getTranslation;
 i18n.addTranslation = i18n.addTranslations;
 
 export { i18n };
+export default i18n;
```
Comment (radekmie) [inline source/common.ts:310]: Why is this needed?

**Review by radekmie — APPROVED** [review]

**Review by zendranm — APPROVED** [review]
